### PR TITLE
removed tags that've only been used once, addresses #255

### PR DESCRIPTION
**Pull request by fat1996**

### CODE REVIEW

File: content/S2015/S15-event-alterconf.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 Title: WiCS is going to AlterConf Toronto! 
 Date: 2015-07-11 9:00
 Category: Events
-Tags: conference
+Tags:
```
Comment (annalorimer) [inline content/S2015/S15-event-alterconf.md]: The conference tag should be kept here.

File: content/S2016/S16-event-square.md [hunk 2]
```diff
@@ -1,7 +1,7 @@
 title: Square Tech Workshop: Becoming Ok with Okhttp
 Date: 2016-05-18
 Category: Events
-Tags: Square, workshop, open source, Okhttp
+Tags: workshop, open source
```
Comment (annalorimer) [inline content/S2016/S16-event-square.md]: Keep the square tag.

File: content/W2016/W16-event-big-data-panel.md [hunk 3]
```diff
@@ -1,7 +1,7 @@
 Title: Big Data Panel
 Date: 2016-02-15
 Category: Events
-Tags: big data, panels
+Tags: panels
```
Comment (annalorimer) [inline content/W2016/W16-event-big-data-panel.md]: I think we should keep tags like 'big data' and 'graphics'. Makes it easy to search for events imo.

File: content/W2016/W16-event-problem-solving-session.md [hunk 4]
```diff
@@ -1,7 +1,7 @@
 title: Problem Solving Session
 Date: 2016-02-01
 Category: Events
-Tags: problem solving, computing, math
+Tags:
```
Comment (annalorimer) [inline content/W2016/W16-event-problem-solving-session.md]: This should probably have the Big CSters tag.

**Review by annalorimer — CHANGES_REQUESTED** [review]

File: content/S2016/S16-event-square.md (hunk 2 above)
Comment (evykassirer) [reply content/S2016/S16-event-square.md]: do we want a tag for every company?

Comment (annalorimer) [reply content/S2016/S16-event-square.md]: Not sure. But there's only two right now so it's not too bad to keep them.

**Comment by fat1996** [comment]
@evykassirer @annalorimer could I get any feedback on this? I wanna close this before the end of this week.

File: content/S2016/S16-event-csters-technical-speaking.md [hunk 5]
```diff
@@ -19,4 +19,4 @@ Note: This event is [women-only]({filename}/pages/faq.md).
 + **Who:** [Women and non-binary individuals]({filename}/pages/faq.md)
 + **What:** Big CSters: Talking About Tech
 + **Where:** QNC 1506
-+ **When:** Monday June 27, 6:00 - 8:00 PM
\ No newline at end of file
++ **When:** Monday June 27, 6:00 - 8:00 PM
```
Comment (evykassirer) [inline content/S2016/S16-event-csters-technical-speaking.md]: may as well change this to ndash

**Comment by evykassirer** [comment]
👍

File: content/S2016/S16-event-csters-technical-speaking.md [hunk 6]
```diff
@@ -19,4 +19,4 @@ Note: This event is [women-only]({filename}/pages/faq.md).
 + **Who:** [Women and non-binary individuals]({filename}/pages/faq.md)
 + **What:** Big CSters: Talking About Tech
 + **Where:** QNC 1506
-+ **When:** Monday June 27, 6:00 - 8:00 PM
\ No newline at end of file
+- **When:** Monday June 27, 6:00 - 8:00 PM
```
Comment (annalorimer) [inline content/S2016/S16-event-csters-technical-speaking.md]: Why was this changed to a '-'?

**Review by annalorimer — COMMENTED** [review]
Just the one question but other than that lgtm.

**Comment by fat1996** [comment]
@annalorimer I misunderstood where the ndash was supposed to go. is this okay?

File: content/S2016/S16-event-csters-technical-speaking.md [hunk 7]
```diff
@@ -19,4 +19,4 @@ Note: This event is [women-only]({filename}/pages/faq.md).
 + **Who:** [Women and non-binary individuals]({filename}/pages/faq.md)
 + **What:** Big CSters: Talking About Tech
 + **Where:** QNC 1506
-- **When:** Monday June 27, 6:00 - 8:00 PM
++ **When:** Monday June 27, 6:00&ndash;8:00 PM
```
Comment (annalorimer) [inline content/S2016/S16-event-csters-technical-speaking.md]: There should only be one '+' :)

Comment (fat1996) [reply content/S2016/S16-event-csters-technical-speaking.md]: @annalorimer i only see 1 '+' in my repo. isn't the second + just showing the addition to the file?

Comment (annalorimer) [reply content/S2016/S16-event-csters-technical-speaking.md]: Yup you're right.

**Review by annalorimer — APPROVED** [review]

**Comment by fat1996** [comment]
@annalorimer @evykassirer can this be merged?

**Pull request merged**